src/clojure/nrepl/misc.clj

```diff
@@ -46,3 +46,15 @@
                                                (-> session meta :id)
                                                session)}))]
     (merge basis response)))
+
+(defn resolve-ns-symbol
```

Comment:
I'd rather keep this private in the ns that uses it, otherwise we're committing to it be part of the API. Maybe we can have a separate ns for useful stuff internally to nREPL that  we can say are not public API.

Reply:
Didn't think about that, gonna add at server ns as private, thanks!